mod-source-record-storage-server/src/main/java/org/folio/verticle/consumers/QuickMarcConsumersVerticle.java

```diff
@@ -0,0 +1,73 @@
+package org.folio.verticle.consumers;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Promise;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.support.AbstractApplicationContext;
+
+import org.folio.dao.util.QMEventTypes;
+import org.folio.kafka.GlobalLoadSensor;
+import org.folio.kafka.KafkaConfig;
+import org.folio.kafka.KafkaConsumerWrapper;
+import org.folio.kafka.KafkaTopicNameHelper;
+import org.folio.kafka.SubscriptionDefinition;
+import org.folio.services.QuickMarcKafkaHandler;
+import org.folio.spring.SpringContextUtil;
+import org.folio.util.pubsub.PubSubClientUtils;
+
+public class QuickMarcConsumersVerticle extends AbstractVerticle {
+
+  private static final GlobalLoadSensor globalLoadSensor = new GlobalLoadSensor();
+
+  private static AbstractApplicationContext springGlobalContext;
+
+  @Autowired
+  private QuickMarcKafkaHandler kafkaHandler;
+
+  @Autowired
+  private KafkaConfig kafkaConfig;
+
+  @Value("${srs.kafka.QuickMarcConsumer.loadLimit:5}")
```

Comment:
please add a note in Readme file that these parameters will be required for the Juniper release. See https://github.com/folio-org/mod-source-record-storage/pull/338

Reply:
added